db/migrate/20150717135909_remove_unique_index_from_builds.rb

```diff
@@ -0,0 +1,7 @@
+class RemoveUniqueIndexFromBuilds < ActiveRecord::Migration
+  def change
+    # Remove unique index on git_sha, users should be able to rebuild docker images against same SHA
```

Comment:
I guess I had thought images could be generated differently depending on environment variable settings without having to touch the github repo. Do the docker projects use the .env stuff? Or are they always going to read everything from Consul? @jonmoter